src/System.Windows.Extensions/tests/System/Drawing/FontConverterTests.cs

```diff
@@ -34,4 +35,114 @@ public void ExTestConvertFrom_ThrowsNotSupportedException()
             Assert.Throws<NotSupportedException>(() => converter.ConvertFrom(1));
         }
     }
+
+    public class FontConverterTest
+    {
+        [ConditionalTheory(Helpers.IsDrawingSupported)]
+        [MemberData(nameof(TestConvertFormData))]
+        public void TestConvertFrom(string input, string name, float size, GraphicsUnit units, FontStyle fontStyle)
```

Comment:
Can you prefix the expected outputs with `expected`, eg `expectedName, expectedSize, etc`